Setup/UI/RequiredSoftwareDlg.wxs

```diff
@@ -21,11 +21,7 @@
         <Control Id="KDiffTitle" Type="Text" Height="12" Width="200" X="15" Y="111" Text="{\WixUI_Font_Title}KDiff3 32bit 0.9.97" />
         <Control Id="KDiffDescription" Type="Text" Height="44" Width="238" X="25" Y="127" Text="KDiff is a diff/merge tool that Git Extensions calls upon when it encounters a merge conflict. If you already have a diff/merge tool that works with Git then you don't need to install this." />
         <Control Id="KDiffCheckBox" Type="CheckBox" Height="20" Width="78" X="274" Y="129" Text="Install KDiff" Property="INSTALLKDIFF3" CheckBoxValue="1" />
-
-        <Control Id="GitCredentialWinStoreTitle" Type="Text" Height="12" Width="200" X="15" Y="176" Text="{\WixUI_Font_Title}Windows Credential Store for Git 1.4" />
```

Comment:
Set Git and KDiff controls coordinates to those that were before commit
https://github.com/karyon/gitextensions/commit/ef379796a25ebea1b10abad13f87e70689840ff4#diff-beae95e27bbb86fbe94b3ea4fb4041f2